File: Econovation_Algorithm_Mentoring/2022코테준비/Programmers/Programmers_FindPrimeNumber.java
public class Programmers_FindPrimeNumber {
    public int solution(int n) {
        int answer = 1;
        l:
        for (int i = 3; i <= n; i++) {
            if (i % 2 == 0) {
                continue l;
            }
            for (int j = 3; j <= Math.sqrt(i); j += 2) {
                if (i % j == 0) {
                    continue l;
                }
            }
            answer++;
        }
        return answer;
    }
}

File: Algorithm_Swift/Algorithm_Swift/Programmers/Programmers_FindPrimeNumber.swift
//
//  Programmers_FindPrimeNumber.swift
//  Algorithm_Swift
//
//  Created by asong on 2022/03/03.
//

import Foundation

func solution(_ n:Int) -> Int {
    var answer: Int = 1
    if n == 2{
        return 1
    }
    l: for i in 3...n{
        if i % 2 == 0 {
            continue;
        }
        var j: Int = 3
        let s = Int(sqrt(Float(i)))
        while j <= s {
            if i % j == 0{
                continue l;
            }
            j += 2
        }
        answer += 1
    }
    return answer
}
